packages/manager/apps/web/client/app/hosting/general-informations/general-informations.constants.js

```diff
@@ -1,7 +1,25 @@
 export const QUOTA_DECIMAL_PRECISION = 2;
 export const CDN_ADVANCED = 'cdn-advanced';
 
+export const HOSTING_NEW_OFFERS = [
+  'HOSTING_STARTER',
+  'HOSTING_STARTER_OVH',
+  'HOSTING_PERSO',
+  'HOSTING_PRO',
+  'HOSTING_PERFORMANCE_1',
+  'HOSTING_PERFORMANCE_2',
+  'HOSTING_PERFORMANCE_3',
+  'HOSTING_PERFORMANCE_4',
+  'HOSTING_FREE_100_M',
+];
```

Comment:
**nitpick:** I already see this data being maintained in `packages/manager/apps/web/client/app/hosting/dashboard/hosting.constant.js`. Can you reuse?

Reply:
Good catch. Fixed!